qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -257,60 +260,37 @@ def test_rmdir_p_dir_with_file(self):
         self.mount_a.stat(self.dir_name)
 
 class TestGetAndPut(TestCephFSShell):
-    # the 'put' command gets tested as well with the 'get' comamnd
-    def test_put_and_get_without_target_directory(self):
-        """
-        Test that put fails without target path
-        """
-        # generate test data in a directory
-        self.run_cephfs_shell_cmd("!mkdir p1")
-        self.run_cephfs_shell_cmd('!dd if=/dev/urandom of=p1/dump1 bs=1M count=1')
-        self.run_cephfs_shell_cmd('!dd if=/dev/urandom of=p1/dump2 bs=2M count=1')
-        self.run_cephfs_shell_cmd('!dd if=/dev/urandom of=p1/dump3 bs=3M count=1')
-
-        # copy the whole directory over to the cephfs
-        o = self.get_cephfs_shell_cmd_output("put p1")
-        log.info("cephfs-shell output:\n{}".format(o))
-
-        # put p1 should pass
-        o = self.mount_a.stat('p1')
-        log.info("mount_a output:\n{}".format(o))
-        o = self.mount_a.stat('p1/dump1')
-        log.info("mount_a output:\n{}".format(o))
-        o = self.mount_a.stat('p1/dump2')
-        log.info("mount_a output:\n{}".format(o))
-        o = self.mount_a.stat('p1/dump3')
-        log.info("mount_a output:\n{}".format(o))
-
-        self.run_cephfs_shell_cmd('!rm -rf p1')
-        o = self.get_cephfs_shell_cmd_output("get p1")
-        o = self.get_cephfs_shell_cmd_output('!stat p1 || echo $?')
-        log.info("cephfs-shell output:\n{}".format(o))
-        self.validate_stat_output(o)
-
-        o = self.get_cephfs_shell_cmd_output('!stat p1/dump1 || echo $?')
-        log.info("cephfs-shell output:\n{}".format(o))
-        self.validate_stat_output(o)
+    def test_without_target_dir(self):
+        """
+        Test that both, put and get commands, without target path.
+        """
+        tempdir = self.mount_a.client_remote.mkdtemp()
+        tempdirname = path.basename(tempdir)
+        dump1, dump2, dump3 = ('dump1', 'dump2', 'dump3')
```

Comment:
Save directly as set instead.

Reply:
Let's skip this? I only have a list of strings and not a list of key-value pairs.

Reply:
True but you are using the same set of strings many times. Instead we can directly use a set or list of thoe string values.
`dump_file = ('dump1', 'dump2', 'dump3')`
Later append tempdirname to dump_file.